docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Troubleshooting.md

```diff
@@ -0,0 +1,64 @@
+# Troubleshooting Common Installation Errors
+
+This guide assumes installation on any supported OS.  
+
+This guide offers support for common issues faced when first installing LORIS. It is assumed that all installation steps provided have been 
+correctly followed and that all required dependencies have been installed.
+
+## Frontend Does Not Load/JavaScript changes not visible
+The first thing to try is to run:
+
+```bash
+make
+```
+
+Alternatively, on development instances:
+
+```bash
+make dev
+```
+
+If that does not solve your issue, this could be because the Config table in the database is not pointing to the correct values. In this case, follow the following steps. 
+
+1. Check the following values in the Config table via the MySQL backend.
+
+   | Name | Label          | Value          |  
+   | ---- | ----------- | ---------- |
+   | base | Base           | /var/www/loris |
```

Comment:
```suggestion
   | base | Base           | $lorisRootDirectory |
```

This is a suggested path in our installation documents but LORIS could be installed somewhere else on the system. If we want to use this value as an example we should add a note like "Assuming you have installed LORIS at the path `/var/www/loris`, ...."